app/javascript/css/main.scss

```diff
@@ -515,3 +515,54 @@ a.disabled {
   color: var(--primary-color);
   border-bottom: 1px dashed var(--primary-color);
 }
+
+.delete-icon {
+  color: var(--status-red);
+}
+
+.inbox-sender {
+  font-size: 16px;
+  font-weight: 600;
+}
+
+.tabs {
+  margin-top: 3ch;
+}
+
+.tab {
```

Comment:
can we use bootstrap for the border rounding, margins, padding, and border?

Reply:
I think I lean leaving them all in the dbh css classes so it's clearly laid out how to get the tab look, but if it's strongly preferred I can change